packages/media-server/srv/media-service.js

```diff
@@ -0,0 +1,82 @@
+module.exports = (srv) => {
+    const loki = require('lokijs');
+    const db = new loki('DB');
```

Comment:
Isn't the whole point of writing custom drivers to be able to write
```javascript
cds.connect.to('loki')
```
?

Reply:
As per the latest recommendation, it was decided not to have custom driver for lokijs.  It will be a standard CAP application that can be used by others to store and read images.